pkg/prometheus/promcfg_test.go

```diff
@@ -2704,3 +2704,35 @@ func TestHonorTimestamps(t *testing.T) {
 		}
 	}
 }
+
+func TestGetSampleLimit(t *testing.T) {
+	tcs := []struct {
+		Enforced uint64
+		Expected uint64
+		User     uint64
+	}{
```

Comment:
I would expect one more case where the user specifies a lower limit than the enforced one

Reply:
Will do but its sort of already covered by:
```
 			Enforced: 0,
			User:     888,
			Expected: 888,
```